test/stringify_test.js

```diff
@@ -113,6 +113,13 @@ suite('ICAL.stringify', function() {
       delete ICAL.design.defaultSet.param.type;
     });
 
+    test('stringify property value containing "escaped" ; , :', function() {
```

Comment:
The part `; , ` does not make sense to me in the description of the test.

Reply:
The proposal to add this test is from 2022, if you consider https://github.com/kewisch/ical.js/pull/535 . It is unclear to me, if any changes I do will be merged in reasonable time, and this is one of the reasons I prefer not to change anything further here.

In RFC 5545 the definitions are:
```
param-value   = paramtext / quoted-string
paramtext     = *SAFE-CHAR
SAFE-CHAR     = WSP / %x21 / %x23-2B / %x2D-39 / %x3C-7E
    / NON-US-ASCII
; Any character except CONTROL, DQUOTE, ";", ":", ","
quoted-string = DQUOTE *QSAFE-CHAR DQUOTE
```
In general, if a parameter value contains ";", ":" or "," then it must be quoted.  In the concrete case, the test does not verify all three cases.


Now, how is the test exactly called is a nuance.  It is a neglectable problem compared to the fact that this change is pending since 2022.  Once some expert decides to merge this code, the expert can rename the test as seen appropriate.

Reply:
You come at the right time, I currently have some eyes on the project and can help get things merged. I think Martin's comment was more about the name of the test, maybe just writing it out, e.g. `stringify property value containing "escaped" semicolons, commas, colons`.

Reply:
Once this is going to be merged, please change the description as seems good.